bears/python/PEP8Bear.py

```diff
@@ -49,6 +49,6 @@ def run(self, filename, file,
 
         for diff in diffs:
             yield Result(self,
-                         "The code does not comply to PEP8.",
+                         "The code does not comply to PyCodeStyle.",
```

Comment:
Honestly we can either discuss this for hours for a semantic issue - or just continue ...